core/src/com/unciv/ui/pickerscreens/TechPickerScreen.kt

```diff
@@ -26,7 +26,7 @@ class TechPickerScreen(internal val civInfo: CivilizationInfo, centerOnTech: Tec
 
     // All these are to counter performance problems when updating buttons for all techs.
     private var researchableTechs = civInfo.gameInfo.ruleSet.technologies.keys
-            .filter { civTech.canBeResearched(it) }.toHashSet()
+            .filter { civTech.canBeResearched(it) && !civTech.isResearched(it) }.toHashSet()
```

Comment:
Why exclude future tech?

Reply:
"Future Tech" is already special enough. There are already written exceptional cases for it in all other places where `researchableTechs` is used: `setButtonsInfo` and `selectTechnologyForFreeTech`, so excluding it here change nothing.

With regard to the usage in the `init`, "Future Tech" always repeats itself as `civInfo.tech.currentTechnology()`, so it does not reach here at all.